src/Object/Product/Product.php

```diff
@@ -197,20 +198,28 @@ class Product extends AbstractObject implements ProductInterface, ValidatableInt
     private $unitPricingUnit;
 
     /**
-     * SKUs / variations
+     * SKUs
      *
-     * @var \Nosto\Object\Product\SkuCollection of SKUs
+     * @var SkuCollection of SKUs
      */
     private $skus;
 
     /**
+     * SKUs
+     *
+     * @var \Nosto\Object\Product\VariationCollection of variations
+     */
+    private $variations;
```

Comment:
Whatever it is that we rename to, I'm okay with that as long as someone carves out time to fix the JS, add tests, fix the product-update API and the tagging-extractor.